network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_index_featured_video_post.html

```diff
@@ -3,14 +3,14 @@
   <div class="tw-grid tw-grid-cols-12 tw-mb-7">
     <div class="tw-pr-[36px] tw-py-3 tw-flex tw-flex-col tw-col-span-12 large:tw-col-span-4 large:tw-py-0">
       <div class="tw-flex tw-flex-wrap">
-        {% include "./blog_card_topics.html" with topics=blog_page.specific.topics.all%}
+        {% include "./blog_topics.html" with topics=blog_page.specific.topics.all%}
```

Comment:
Think there is a missing space here before the closing %

```suggestion
        {% include "./blog_topics.html" with topics=blog_page.specific.topics.all %}
```